src/components/SharingTags.js

```diff
@@ -0,0 +1,54 @@
+import startCase from 'lodash.startcase'
+import PropTypes from 'prop-types'
+import React from 'react'
+import Helmet from 'react-helmet'
+import { connect } from 'react-redux'
+
+import offenses from '../util/offenses'
+import { getAgency } from '../util/ori'
+import { getPlaceInfo } from '../util/place'
+import usa from '../util/usa'
+
+const SharingTags = ({ agency, crime, place, since, until }) => {
```

Comment:
does this play nice with the other <Helmet> components that are present in some of our parent components (i.e., the ones that set the page titles)?